tasks/35_array-leaders/array-leaders.js

```diff
@@ -0,0 +1,14 @@
+export function arrayLeaders(arr) {
+  let sum = 0;
+  const res = [];
+  for (let i=0; i<arr.length; i++) {
+    for (let j=i+1; j< arr.length; j++) {
```

Comment:
I think that for in for could be avoided here if you start from the other end of the array.

Reply:
You are right again, I optimized it so it only uses one `for` loop.